modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/handler/render/ComponentHandlerWorker.java

```diff
@@ -159,4 +162,8 @@ private Component getFragmentComponent( final FragmentComponent component )
         }
     }
 
+    private PortalResponse addHasContributionsHeader( final PortalResponse source) {
+        return PortalResponse.create(source).header( "Has-Contributions", "true" ).build();
```

Comment:
```suggestion
        return PortalResponse.create(source).header( "X-Has-Contributions", "true" ).build();
```

Reply:
My first thought was also to use `X-` but there's a catch:
https://datatracker.ietf.org/doc/html/rfc6648

Reply:
Then, according to this RFC it should have a prefix `Enonic-`. 
I prefer `X-` though - because static analysis tools are picky and warn about unknown headers, if they don't start with `X-`

Reply:
Let's stick to `X-` then